### PR TITLE
Support ember 2.14 and less

### LINKED ISSUE #32: Incompatibility with Ember < 2.15
Hi @Exelord,

I was working through testing PR #4, and encountered a build issue.  I traced the issue back to the `app.import('node_modules/rollbar/dist/rollbar.named-amd.js')` that's being done in `index.js`.  Ember versions prior to 2.15 do *not* support `app.import` for `node_modules` dependencies.  

Do you still want to support these older versions going forward?  The README shows >= 2.8 should work.  My project is currently on 2.13.  I think to resolve this particular issue, these changes would have to be reversed: https://github.com/Exelord/ember-rollbar-client/commit/6e9e7e29657726bbba883f90f5117c9410273d8f#diff-168726dbe96b3ce427e7fedce31bb0bc

### PR DESCRIPTION
**Pull request by svvitale**
Fixes #32 by removing the `app.import` for the rollbar client.  Also includes the map file if sourcemaps are enabled which resolves another warning.

### CODE REVIEW

**Comment by svvitale** [comment]
@Exelord I think CI is failing because of a borked package cache (it's only failing on Ember 2.12 currently).  Can you flush the cache and retry the build for this PR?

File: .travis.yml [hunk 1]
```diff
@@ -36,9 +36,12 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g npm@4
+  - npm install -g npm@latest
```
Comment (Exelord) [inline .travis.yml]: it should be 4 as it was :/

File: .travis.yml [hunk 2]
```diff
@@ -36,9 +36,12 @@ matrix:
...
   - npm --version
 
+install:
```
Comment (Exelord) [inline .travis.yml]: Why? Ember cli doesn't add this change

File: .travis.yml [hunk 3]
```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "4"
```
Comment (Exelord) [inline .travis.yml]: it should be `4`
https://github.com/ember-cli/ember-addon-output/blob/v3.0.0/.travis.yml

**Comment by svvitale** [comment]
Removed all travis changes.  Can you try to re-run the CI build?  The failures don't appear to be related to this change.

**Pull request merged**